docker/run.sh

```diff
@@ -72,8 +72,8 @@ reload)
     bundle exec rake db:create 
     echo "executing rake db:schema:load"
     bundle exec rake db:schema:load
-    echo "executing rake db:data:load"
-    bundle exec rake db:data:load
+    echo "executing rake claims:restore"
+    bundle exec rake claims:restore
     ;;
 reseed)
     # db:clear
```

Comment:
You're going to kill me Ben, but I think we're going to drop the idea of dumping data and restoring, and go back to the old idea of re-seeding the database each deploy.